state/executor.go

```diff
@@ -365,7 +373,8 @@ func (e *BlockExecutor) execute(ctx context.Context, state types.State, block *t
 		Misbehavior: abciBlock.Evidence.Evidence.ToABCI(),
 		Txs:         abciBlock.Txs.ToSliceOfBytes(),
 	})
-
+	endTime := time.Now().UnixNano()
+	e.metrics.BlockProcessingTime.Observe(float64(endTime-startTime) / 1000000)
```

Comment:
We should either use a constant here or at least document what dividing by`1000000` does. What does dividing time in nanoseconds by `1000000` do?

Reply:
this is directly copied from cometbft. the constant is for converting from nanoseconds to milliseconds which makes sense as metrics generally do ms.